admin/sql/updates/2021-08-28-add-user-timeline-enum-type-critiquebrainz_review.sql

```diff
@@ -0,0 +1,5 @@
+BEGIN;
```

Comment:
You can remove the BEGIN/COMMIT here, Postgres is weird about ALTER TYPE ADD VALUE running inside a transaction.